inst/WORDLIST

```diff
@@ -8,3 +8,5 @@ funder
 pre
 repo
 shinyServer
+gg
+ggplot
```

Comment:
please change documentation so that they don't appear here. Please use backticks instead of quotation for description of R objects, classes, functions, packages